lib/rest.js

```diff
@@ -33,6 +34,9 @@ module.exports = class MicroStrategyRESTAPI extends RestConnection {
     // https://demo.microstrategy.com/MicroStrategyLibrary/api-docs/index.html#/User%20Management
     this.userManagement = new UserManagementAPI(this);
 
+    // https://demo.microstrategy.com/MicroStrategyLibrary/api-docs/index.html#/Library
+    this.library = new LibraryAPI(this);
```

Comment:
Move this up please, around line 27 (after dossiersAndDocuments, before objectManagement).

Reply:
done